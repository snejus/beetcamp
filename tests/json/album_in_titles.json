{
  "@context": "https://schema.org",
  "@id": "https://gutterfunkuk.bandcamp.com/album/gutterfunk-all-subject-to-vibes-various-artists-lp",
  "@type": "MusicAlbum",
  "additionalProperty": [
    {
      "@type": "PropertyValue",
      "name": "art_id",
      "value": 141370490
    },
    {
      "@type": "PropertyValue",
      "name": "featured_track_num",
      "value": 6
    },
    {
      "@type": "PropertyValue",
      "name": "license_name",
      "value": "all_rights_reserved"
    }
  ],
  "albumRelease": [
    {
      "@id": "https://gutterfunkuk.bandcamp.com/album/gutterfunk-all-subject-to-vibes-various-artists-lp",
      "@type": [
        "MusicRelease",
        "Product"
      ],
      "additionalProperty": [
        {
          "@type": "PropertyValue",
          "name": "item_id",
          "value": 744405404
        },
        {
          "@type": "PropertyValue",
          "name": "item_type",
          "value": "a"
        },
        {
          "@type": "PropertyValue",
          "name": "selling_band_id",
          "value": 996126859
        },
        {
          "@type": "PropertyValue",
          "name": "type_name",
          "value": "Digital"
        },
        {
          "@type": "PropertyValue",
          "name": "art_id",
          "value": 141370490
        }
      ],
      "description": "Includes high-quality download in MP3, FLAC and more. Paying supporters also get unlimited streaming via the free Bandcamp app.",
      "image": [
        "https://f4.bcbits.com/img/a0141370490_10.jpg"
      ],
      "musicReleaseFormat": "DigitalFormat",
      "name": "GutterFunk: All Subject To Vibes [Various Artists LP]"
    },
    {
      "@id": "https://gutterfunkuk.bandcamp.com/album/gutterfunk-all-subject-to-vibes-various-artists-lp#b56755260",
      "@type": [
        "MusicRelease",
        "Product"
      ],
      "additionalProperty": [
        {
          "@type": "PropertyValue",
          "name": "item_id",
          "value": 56755260
        },
        {
          "@type": "PropertyValue",
          "name": "item_type",
          "value": "b"
        },
        {
          "@type": "PropertyValue",
          "name": "selling_band_id",
          "value": 996126859
        },
        {
          "@type": "PropertyValue",
          "name": "type_name",
          "value": "Digital"
        },
        {
          "@type": "PropertyValue",
          "name": "art_id",
          "value": 4294573575
        },
        {
          "@type": "PropertyValue",
          "name": "is_bfd",
          "value": true
        }
      ],
      "image": [
        "https://f4.bcbits.com/img/a4294573575_10.jpg"
      ],
      "musicReleaseFormat": "DigitalFormat",
      "name": "full digital discography (49 releases)",
      "offers": {
        "@type": "Offer",
        "additionalProperty": [
          {
            "@type": "PropertyValue",
            "name": "bundle_size",
            "value": 49
          },
          {
            "@type": "PropertyValue",
            "name": "discount",
            "value": 0.35
          }
        ],
        "availability": "OnlineOnly",
        "price": 102.05,
        "priceCurrency": "GBP",
        "priceSpecification": {
          "minPrice": 102.05
        },
        "url": "https://gutterfunkuk.bandcamp.com/album/gutterfunk-all-subject-to-vibes-various-artists-lp#b56755260-buy"
      }
    },
    {
      "@id": "https://gutterfunkuk.bandcamp.com/track/diemantle-spirit-world",
      "@type": "MusicRelease"
    },
    {
      "@id": "https://gutterfunkuk.bandcamp.com/track/sure-thing-special-love-nuff-pedals-remix-all-subject-to-vibes-lp-gutterfunk",
      "@type": "MusicRelease"
    },
    {
      "@id": "https://gutterfunkuk.bandcamp.com/track/monkey-wrench-lakossa-all-subject-to-vibes-lp-gutterfunk",
      "@type": "MusicRelease"
    },
    {
      "@id": "https://gutterfunkuk.bandcamp.com/track/addison-groove-tokyo-beat-all-subject-to-vibes-lp-gutterfunk",
      "@type": "MusicRelease"
    },
    {
      "@id": "https://gutterfunkuk.bandcamp.com/track/fox-diemantle-more-all-subject-to-vibes-lp-gutterfunk",
      "@type": "MusicRelease"
    },
    {
      "@id": "https://gutterfunkuk.bandcamp.com/track/dj-die-pinch-jungle-reflection-all-subject-to-vibes-lp-gutterfunk",
      "@type": "MusicRelease"
    },
    {
      "@id": "https://gutterfunkuk.bandcamp.com/track/dismantle-bodied-all-subject-to-vibes-lp-gutterfunk",
      "@type": "MusicRelease"
    },
    {
      "@id": "https://gutterfunkuk.bandcamp.com/track/dead-mans-chest-josimar-out-there-all-subject-to-vibes-lp-gutterfunk",
      "@type": "MusicRelease"
    },
    {
      "@id": "https://gutterfunkuk.bandcamp.com/track/fixate-one-for-the-floor-all-subject-to-vibes-lp-gutterfunk",
      "@type": "MusicRelease"
    },
    {
      "@id": "https://gutterfunkuk.bandcamp.com/track/watch-the-ride-dj-randall-dj-die-dismantle-skeematics-all-subject-to-vibes-lp-gutterfunk",
      "@type": "MusicRelease"
    },
    {
      "@id": "https://gutterfunkuk.bandcamp.com/track/sure-thing-atoms-all-subject-to-vibes-lp-gutterfunk",
      "@type": "MusicRelease"
    }
  ],
  "albumReleaseType": "AlbumRelease",
  "byArtist": {
    "@id": "https://gutterfunkuk.bandcamp.com",
    "@type": "MusicGroup",
    "name": "GutterFunk"
  },
  "copyrightNotice": "All Rights Reserved",
  "dateModified": "13 Dec 2019 08:17:36 GMT",
  "datePublished": "13 Dec 2019 00:00:00 GMT",
  "description": "DJ Die's GutterFunk imprint has gone from strength to strength over the past few years, the label's ethos of ‘good-music-first’, whilst not being restricted by genre, has permeated with artists and listeners alike. The labels tracks have found their way into the sets of a massively diverse set of DJs - from the likes of Gerd Janson, David Rodigan, dBridge and Shy FX.\r\n\r\nThe labels first compilation album, ‘GutterFunk: All Subject To Vibes’ acts as an introduction or indeed a reminder of what GutterFunk is all about. Style-wise we are presented with House, mutations of 140bpm, Broken Beat, Drum and Bass, Jungle and Soul. The compilation is encapsulated by an archive photo from the legendary photographer Mark Simmons, reflecting the curveball nature of the album, whilst also paying homage to Bristol values of ignoring rules and tradition, which is embedded into GutterFunk’s DNA. The artwork used for the album is a photo by Mark Simmons from his book 'Bristol Black & White'.\r\n\r\n The guest list on the album speaks for itself, showing GutterFunk’s penchant for high-quality artists: DJ Die, Pinch, Fox, Randall, Dismantle, Addison Groove and Fixate to name a few. Meanwhile the newly formed Watch The Ride project delivers the club anthem 'Skeematics', riding high after the massive success of the collab with D Double E and mysterious Soul/boogie outfit Sure Thing close off the album with the perfect last dance 'Atoms.'\r\n\r\nIn true GutterFunk fashion, the album was worked on with a fine-tooth comb, each detail in each track was agonised over by the artist and then by DJ Die and Dismantle emphasising the importance of the labels first compilation album. It's an album that prides itself on working as a complete listen when going from track 1 to track 11, whilst working as DJ tools when used as club tracks.\r\n\r\nA great way to end a big year for the label and celebrate the achievements of all the artists’ successes during 2019.",
  "image": "https://f4.bcbits.com/img/a0141370490_10.jpg",
  "keywords": [
    "Bass",
    "Boogie",
    "Broken Beat",
    "Electronic",
    "House",
    "Jungle",
    "disco",
    "drum & bass",
    "dub",
    "Bristol"
  ],
  "mainEntityOfPage": "https://gutterfunkuk.bandcamp.com/album/gutterfunk-all-subject-to-vibes-various-artists-lp",
  "name": "GutterFunk: All Subject To Vibes [Various Artists LP]",
  "numTracks": 11,
  "publisher": {
    "@id": "https://gutterfunkuk.bandcamp.com",
    "@type": "MusicGroup",
    "additionalProperty": [
      {
        "@type": "PropertyValue",
        "name": "band_id",
        "value": 996126859
      },
      {
        "@type": "PropertyValue",
        "name": "has_any_downloads",
        "value": true
      },
      {
        "@type": "PropertyValue",
        "name": "has_policies",
        "value": true
      },
      {
        "@type": "PropertyValue",
        "name": "image_height",
        "value": 500
      },
      {
        "@type": "PropertyValue",
        "name": "image_id",
        "value": 10127893
      },
      {
        "@type": "PropertyValue",
        "name": "image_width",
        "value": 500
      }
    ],
    "description": "Bristol's musical pioneer DJ Die presents GutterFunk.\n\nBristol has always been a melting pot full of musical styles & GutterFunk stays true to the rich sonic heritage of the city. \n\nOur roots are in Bristol but our branches spread worldwide.",
    "foundingLocation": {
      "@type": "Place",
      "name": "Bristol, UK"
    },
    "genre": "https://bandcamp.com/discover/electronic",
    "image": "https://f4.bcbits.com/img/0010127893_10.jpg",
    "mainEntityOfPage": [
      {
        "@type": "WebSite",
        "name": "gutterfunk.co",
        "url": "http://www.gutterfunk.co"
      },
      {
        "@type": "WebPage",
        "name": "SoundCloud",
        "url": "https://soundcloud.com/gutterfunk"
      },
      {
        "@type": "WebPage",
        "name": "Facebook",
        "url": "https://www.facebook.com/Gutterfunk"
      }
    ],
    "name": "GutterFunk",
    "subjectOf": [
      {
        "@type": "WebPage",
        "additionalProperty": [
          {
            "@type": "PropertyValue",
            "name": "nav_type",
            "value": "m"
          }
        ],
        "name": "Music",
        "url": "https://gutterfunkuk.bandcamp.com/music"
      },
      {
        "@type": "WebPage",
        "additionalProperty": [
          {
            "@type": "PropertyValue",
            "name": "nav_type",
            "value": "p"
          }
        ],
        "name": "Store",
        "url": "https://gutterfunkuk.bandcamp.com/merch"
      },
      {
        "@type": "WebPage",
        "additionalProperty": [
          {
            "@type": "PropertyValue",
            "name": "nav_type",
            "value": "c"
          }
        ],
        "name": "community",
        "url": "https://gutterfunkuk.bandcamp.com/community"
      }
    ]
  },
  "track": {
    "@type": "ItemList",
    "itemListElement": [
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://gutterfunkuk.bandcamp.com/track/diemantle-spirit-world",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 1393548334
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "all_rights_reserved"
            }
          ],
          "copyrightNotice": "All Rights Reserved",
          "duration": "P00H02M24S",
          "mainEntityOfPage": "https://gutterfunkuk.bandcamp.com/track/diemantle-spirit-world",
          "name": "DieMantle- Spirit World"
        },
        "position": 1
      },
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://gutterfunkuk.bandcamp.com/track/sure-thing-special-love-nuff-pedals-remix-all-subject-to-vibes-lp-gutterfunk",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 3689168236
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "all_rights_reserved"
            }
          ],
          "copyrightNotice": "All Rights Reserved",
          "duration": "P00H04M46S",
          "mainEntityOfPage": "https://gutterfunkuk.bandcamp.com/track/sure-thing-special-love-nuff-pedals-remix-all-subject-to-vibes-lp-gutterfunk",
          "name": "Sure Thing - Special Love (Nuff Pedals Remix) -  [All Subject To Vibes LP] - GutterFunk"
        },
        "position": 2
      },
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://gutterfunkuk.bandcamp.com/track/monkey-wrench-lakossa-all-subject-to-vibes-lp-gutterfunk",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 1202933604
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "all_rights_reserved"
            }
          ],
          "copyrightNotice": "All Rights Reserved",
          "duration": "P00H06M24S",
          "mainEntityOfPage": "https://gutterfunkuk.bandcamp.com/track/monkey-wrench-lakossa-all-subject-to-vibes-lp-gutterfunk",
          "name": "Monkey Wrench - Lakossa -  [All Subject To Vibes LP] - GutterFunk"
        },
        "position": 3
      },
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://gutterfunkuk.bandcamp.com/track/addison-groove-tokyo-beat-all-subject-to-vibes-lp-gutterfunk",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 1204145258
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "all_rights_reserved"
            }
          ],
          "copyrightNotice": "All Rights Reserved",
          "duration": "P00H03M44S",
          "mainEntityOfPage": "https://gutterfunkuk.bandcamp.com/track/addison-groove-tokyo-beat-all-subject-to-vibes-lp-gutterfunk",
          "name": "Addison Groove - Tokyo Beat -  [All Subject To Vibes LP] - GutterFunk"
        },
        "position": 4
      },
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://gutterfunkuk.bandcamp.com/track/fox-diemantle-more-all-subject-to-vibes-lp-gutterfunk",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 2673057261
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "all_rights_reserved"
            }
          ],
          "copyrightNotice": "All Rights Reserved",
          "duration": "P00H04M00S",
          "mainEntityOfPage": "https://gutterfunkuk.bandcamp.com/track/fox-diemantle-more-all-subject-to-vibes-lp-gutterfunk",
          "name": "Fox & DieMantle - More -  [All Subject To Vibes LP] - GutterFunk"
        },
        "position": 5
      },
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://gutterfunkuk.bandcamp.com/track/dj-die-pinch-jungle-reflection-all-subject-to-vibes-lp-gutterfunk",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 3165824141
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "all_rights_reserved"
            }
          ],
          "copyrightNotice": "All Rights Reserved",
          "duration": "P00H04M19S",
          "mainEntityOfPage": "https://gutterfunkuk.bandcamp.com/track/dj-die-pinch-jungle-reflection-all-subject-to-vibes-lp-gutterfunk",
          "name": "DJ Die & Pinch - Jungle Reflection -  [All Subject To Vibes LP] - GutterFunk"
        },
        "position": 6
      },
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://gutterfunkuk.bandcamp.com/track/dismantle-bodied-all-subject-to-vibes-lp-gutterfunk",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 360864791
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "all_rights_reserved"
            }
          ],
          "copyrightNotice": "All Rights Reserved",
          "duration": "P00H03M06S",
          "mainEntityOfPage": "https://gutterfunkuk.bandcamp.com/track/dismantle-bodied-all-subject-to-vibes-lp-gutterfunk",
          "name": "Dismantle - Bodied -  [All Subject To Vibes LP] - GutterFunk"
        },
        "position": 7
      },
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://gutterfunkuk.bandcamp.com/track/dead-mans-chest-josimar-out-there-all-subject-to-vibes-lp-gutterfunk",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 3133724441
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "all_rights_reserved"
            }
          ],
          "copyrightNotice": "All Rights Reserved",
          "duration": "P00H05M44S",
          "mainEntityOfPage": "https://gutterfunkuk.bandcamp.com/track/dead-mans-chest-josimar-out-there-all-subject-to-vibes-lp-gutterfunk",
          "name": "Dead Mans Chest & Josimar - Out There -  [All Subject To Vibes LP]- GutterFunk"
        },
        "position": 8
      },
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://gutterfunkuk.bandcamp.com/track/fixate-one-for-the-floor-all-subject-to-vibes-lp-gutterfunk",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 366977680
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "all_rights_reserved"
            }
          ],
          "copyrightNotice": "All Rights Reserved",
          "duration": "P00H06M26S",
          "mainEntityOfPage": "https://gutterfunkuk.bandcamp.com/track/fixate-one-for-the-floor-all-subject-to-vibes-lp-gutterfunk",
          "name": "Fixate - One For The Floor -  [All Subject To Vibes LP] - GutterFunk"
        },
        "position": 9
      },
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://gutterfunkuk.bandcamp.com/track/watch-the-ride-dj-randall-dj-die-dismantle-skeematics-all-subject-to-vibes-lp-gutterfunk",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 3110645604
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "all_rights_reserved"
            }
          ],
          "copyrightNotice": "All Rights Reserved",
          "duration": "P00H04M24S",
          "mainEntityOfPage": "https://gutterfunkuk.bandcamp.com/track/watch-the-ride-dj-randall-dj-die-dismantle-skeematics-all-subject-to-vibes-lp-gutterfunk",
          "name": "Watch The Ride (DJ Randall, DJ Die, Dismantle)  - Skeematics -  [All Subject To Vibes LP] - GutterFunk"
        },
        "position": 10
      },
      {
        "@type": "ListItem",
        "item": {
          "@id": "https://gutterfunkuk.bandcamp.com/track/sure-thing-atoms-all-subject-to-vibes-lp-gutterfunk",
          "@type": "MusicRecording",
          "additionalProperty": [
            {
              "@type": "PropertyValue",
              "name": "track_id",
              "value": 1590363110
            },
            {
              "@type": "PropertyValue",
              "name": "license_name",
              "value": "all_rights_reserved"
            }
          ],
          "copyrightNotice": "All Rights Reserved",
          "duration": "P00H07M34S",
          "mainEntityOfPage": "https://gutterfunkuk.bandcamp.com/track/sure-thing-atoms-all-subject-to-vibes-lp-gutterfunk",
          "name": "Sure Thing - Atoms - [All Subject To Vibes LP] - GutterFunk"
        },
        "position": 11
      }
    ],
    "numberOfItems": 11
  }
}
