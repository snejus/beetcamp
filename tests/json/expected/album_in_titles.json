[{"album": "All Subject To Vibes LP", "album_id": "https://gutterfunkuk.bandcamp.com/album/gutterfunk-all-subject-to-vibes-various-artists-lp", "artist": "Various Artists", "artist_id": "https://gutterfunkuk.bandcamp.com", "tracks": [{"title": "Spirit World", "track_id": "https://gutterfunkuk.bandcamp.com/track/diemantle-spirit-world", "release_track_id": null, "artist": "DieMantle", "artist_id": "https://gutterfunkuk.bandcamp.com", "length": 144, "index": 1, "media": "Digital Media", "medium": 1, "medium_index": 1, "medium_total": 11, "artist_sort": null, "disctitle": null, "artist_credit": null, "data_source": "bandcamp", "data_url": "https://gutterfunkuk.bandcamp.com/album/gutterfunk-all-subject-to-vibes-various-artists-lp", "lyricist": null, "composer": null, "composer_sort": null, "arranger": null, "track_alt": null, "work": null, "mb_workid": null, "work_disambig": null, "bpm": null, "initial_key": null, "genre": null}, {"title": "Special Love (Nuff Pedals Remix)", "track_id": "https://gutterfunkuk.bandcamp.com/track/sure-thing-special-love-nuff-pedals-remix-all-subject-to-vibes-lp-gutterfunk", "release_track_id": null, "artist": "Sure Thing", "artist_id": "https://gutterfunkuk.bandcamp.com", "length": 286, "index": 2, "media": "Digital Media", "medium": 1, "medium_index": 2, "medium_total": 11, "artist_sort": null, "disctitle": null, "artist_credit": null, "data_source": "bandcamp", "data_url": "https://gutterfunkuk.bandcamp.com/album/gutterfunk-all-subject-to-vibes-various-artists-lp", "lyricist": null, "composer": null, "composer_sort": null, "arranger": null, "track_alt": null, "work": null, "mb_workid": null, "work_disambig": null, "bpm": null, "initial_key": null, "genre": null}, {"title": "Lakossa", "track_id": "https://gutterfunkuk.bandcamp.com/track/monkey-wrench-lakossa-all-subject-to-vibes-lp-gutterfunk", "release_track_id": null, "artist": "Monkey Wrench", "artist_id": "https://gutterfunkuk.bandcamp.com", "length": 384, "index": 3, "media": "Digital Media", "medium": 1, "medium_index": 3, "medium_total": 11, "artist_sort": null, "disctitle": null, "artist_credit": null, "data_source": "bandcamp", "data_url": "https://gutterfunkuk.bandcamp.com/album/gutterfunk-all-subject-to-vibes-various-artists-lp", "lyricist": null, "composer": null, "composer_sort": null, "arranger": null, "track_alt": null, "work": null, "mb_workid": null, "work_disambig": null, "bpm": null, "initial_key": null, "genre": null}, {"title": "Tokyo Beat", "track_id": "https://gutterfunkuk.bandcamp.com/track/addison-groove-tokyo-beat-all-subject-to-vibes-lp-gutterfunk", "release_track_id": null, "artist": "Addison Groove", "artist_id": "https://gutterfunkuk.bandcamp.com", "length": 224, "index": 4, "media": "Digital Media", "medium": 1, "medium_index": 4, "medium_total": 11, "artist_sort": null, "disctitle": null, "artist_credit": null, "data_source": "bandcamp", "data_url": "https://gutterfunkuk.bandcamp.com/album/gutterfunk-all-subject-to-vibes-various-artists-lp", "lyricist": null, "composer": null, "composer_sort": null, "arranger": null, "track_alt": null, "work": null, "mb_workid": null, "work_disambig": null, "bpm": null, "initial_key": null, "genre": null}, {"title": "More", "track_id": "https://gutterfunkuk.bandcamp.com/track/fox-diemantle-more-all-subject-to-vibes-lp-gutterfunk", "release_track_id": null, "artist": "Fox & DieMantle", "artist_id": "https://gutterfunkuk.bandcamp.com", "length": 240, "index": 5, "media": "Digital Media", "medium": 1, "medium_index": 5, "medium_total": 11, "artist_sort": null, "disctitle": null, "artist_credit": null, "data_source": "bandcamp", "data_url": "https://gutterfunkuk.bandcamp.com/album/gutterfunk-all-subject-to-vibes-various-artists-lp", "lyricist": null, "composer": null, "composer_sort": null, "arranger": null, "track_alt": null, "work": null, "mb_workid": null, "work_disambig": null, "bpm": null, "initial_key": null, "genre": null}, {"title": "Jungle Reflection", "track_id": "https://gutterfunkuk.bandcamp.com/track/dj-die-pinch-jungle-reflection-all-subject-to-vibes-lp-gutterfunk", "release_track_id": null, "artist": "DJ Die & Pinch", "artist_id": "https://gutterfunkuk.bandcamp.com", "length": 259, "index": 6, "media": "Digital Media", "medium": 1, "medium_index": 6, "medium_total": 11, "artist_sort": null, "disctitle": null, "artist_credit": null, "data_source": "bandcamp", "data_url": "https://gutterfunkuk.bandcamp.com/album/gutterfunk-all-subject-to-vibes-various-artists-lp", "lyricist": null, "composer": null, "composer_sort": null, "arranger": null, "track_alt": null, "work": null, "mb_workid": null, "work_disambig": null, "bpm": null, "initial_key": null, "genre": null}, {"title": "Bodied", "track_id": "https://gutterfunkuk.bandcamp.com/track/dismantle-bodied-all-subject-to-vibes-lp-gutterfunk", "release_track_id": null, "artist": "Dismantle", "artist_id": "https://gutterfunkuk.bandcamp.com", "length": 186, "index": 7, "media": "Digital Media", "medium": 1, "medium_index": 7, "medium_total": 11, "artist_sort": null, "disctitle": null, "artist_credit": null, "data_source": "bandcamp", "data_url": "https://gutterfunkuk.bandcamp.com/album/gutterfunk-all-subject-to-vibes-various-artists-lp", "lyricist": null, "composer": null, "composer_sort": null, "arranger": null, "track_alt": null, "work": null, "mb_workid": null, "work_disambig": null, "bpm": null, "initial_key": null, "genre": null}, {"title": "Out There", "track_id": "https://gutterfunkuk.bandcamp.com/track/dead-mans-chest-josimar-out-there-all-subject-to-vibes-lp-gutterfunk", "release_track_id": null, "artist": "Dead Mans Chest & Josimar", "artist_id": "https://gutterfunkuk.bandcamp.com", "length": 344, "index": 8, "media": "Digital Media", "medium": 1, "medium_index": 8, "medium_total": 11, "artist_sort": null, "disctitle": null, "artist_credit": null, "data_source": "bandcamp", "data_url": "https://gutterfunkuk.bandcamp.com/album/gutterfunk-all-subject-to-vibes-various-artists-lp", "lyricist": null, "composer": null, "composer_sort": null, "arranger": null, "track_alt": null, "work": null, "mb_workid": null, "work_disambig": null, "bpm": null, "initial_key": null, "genre": null}, {"title": "One For The Floor", "track_id": "https://gutterfunkuk.bandcamp.com/track/fixate-one-for-the-floor-all-subject-to-vibes-lp-gutterfunk", "release_track_id": null, "artist": "Fixate", "artist_id": "https://gutterfunkuk.bandcamp.com", "length": 386, "index": 9, "media": "Digital Media", "medium": 1, "medium_index": 9, "medium_total": 11, "artist_sort": null, "disctitle": null, "artist_credit": null, "data_source": "bandcamp", "data_url": "https://gutterfunkuk.bandcamp.com/album/gutterfunk-all-subject-to-vibes-various-artists-lp", "lyricist": null, "composer": null, "composer_sort": null, "arranger": null, "track_alt": null, "work": null, "mb_workid": null, "work_disambig": null, "bpm": null, "initial_key": null, "genre": null}, {"title": "Skeematics", "track_id": "https://gutterfunkuk.bandcamp.com/track/watch-the-ride-dj-randall-dj-die-dismantle-skeematics-all-subject-to-vibes-lp-gutterfunk", "release_track_id": null, "artist": "Watch The Ride (DJ Randall, DJ Die, Dismantle)", "artist_id": "https://gutterfunkuk.bandcamp.com", "length": 264, "index": 10, "media": "Digital Media", "medium": 1, "medium_index": 10, "medium_total": 11, "artist_sort": null, "disctitle": null, "artist_credit": null, "data_source": "bandcamp", "data_url": "https://gutterfunkuk.bandcamp.com/album/gutterfunk-all-subject-to-vibes-various-artists-lp", "lyricist": null, "composer": null, "composer_sort": null, "arranger": null, "track_alt": null, "work": null, "mb_workid": null, "work_disambig": null, "bpm": null, "initial_key": null, "genre": null}, {"title": "Atoms", "track_id": "https://gutterfunkuk.bandcamp.com/track/sure-thing-atoms-all-subject-to-vibes-lp-gutterfunk", "release_track_id": null, "artist": "Sure Thing", "artist_id": "https://gutterfunkuk.bandcamp.com", "length": 454, "index": 11, "media": "Digital Media", "medium": 1, "medium_index": 11, "medium_total": 11, "artist_sort": null, "disctitle": null, "artist_credit": null, "data_source": "bandcamp", "data_url": "https://gutterfunkuk.bandcamp.com/album/gutterfunk-all-subject-to-vibes-various-artists-lp", "lyricist": null, "composer": null, "composer_sort": null, "arranger": null, "track_alt": null, "work": null, "mb_workid": null, "work_disambig": null, "bpm": null, "initial_key": null, "genre": null}], "asin": null, "albumtype": "album", "va": true, "year": 2019, "month": 12, "day": 13, "label": "GutterFunk", "mediums": 1, "artist_sort": null, "releasegroup_id": null, "catalognum": "", "script": null, "language": null, "country": "GB", "style": "electronic", "genre": "broken beat, disco, drum and bass, dub, house, jungle", "albumstatus": "Official", "media": "Digital Media", "albumdisambig": null, "releasegroupdisambig": null, "artist_credit": null, "original_year": null, "original_month": null, "original_day": null, "data_source": "bandcamp", "data_url": "https://gutterfunkuk.bandcamp.com/album/gutterfunk-all-subject-to-vibes-various-artists-lp", "discogs_albumid": null, "discogs_labelid": null, "discogs_artistid": null, "albumtypes": "album; lp", "comments": "DJ Die's GutterFunk imprint has gone from strength to strength over the past few years, the label's ethos of \u2018good-music-first\u2019, whilst not being restricted by genre, has permeated with artists and listeners alike. The labels tracks have found their way into the sets of a massively diverse set of DJs - from the likes of Gerd Janson, David Rodigan, dBridge and Shy FX.\n\nThe labels first compilation album, \u2018GutterFunk: All Subject To Vibes\u2019 acts as an introduction or indeed a reminder of what GutterFunk is all about. Style-wise we are presented with House, mutations of 140bpm, Broken Beat, Drum and Bass, Jungle and Soul. The compilation is encapsulated by an archive photo from the legendary photographer Mark Simmons, reflecting the curveball nature of the album, whilst also paying homage to Bristol values of ignoring rules and tradition, which is embedded into GutterFunk\u2019s DNA. The artwork used for the album is a photo by Mark Simmons from his book 'Bristol Black & White'.\n\n The guest list on the album speaks for itself, showing GutterFunk\u2019s penchant for high-quality artists: DJ Die, Pinch, Fox, Randall, Dismantle, Addison Groove and Fixate to name a few. Meanwhile the newly formed Watch The Ride project delivers the club anthem 'Skeematics', riding high after the massive success of the collab with D Double E and mysterious Soul/boogie outfit Sure Thing close off the album with the perfect last dance 'Atoms.'\n\nIn true GutterFunk fashion, the album was worked on with a fine-tooth comb, each detail in each track was agonised over by the artist and then by DJ Die and Dismantle emphasising the importance of the labels first compilation album. It's an album that prides itself on working as a complete listen when going from track 1 to track 11, whilst working as DJ tools when used as club tracks.\n\nA great way to end a big year for the label and celebrate the achievements of all the artists\u2019 successes during 2019."}]
